File: MealMaster/MealMaster.BLL/DTOs/Response/Menu/MenuResponseDto.cs
using MealMaster.BLL.DTOs.Request.MenuItem;
using MealMaster.BLL.DTOs.Response.MenuItem;

namespace MealMaster.BLL.DTOs.Response.Menu;

public class MenuResponseDto
{
    public Guid UserId { get; set; }
    public DateTime Date { get; set; }
    public int TotalCalories { get; set; }
    public int MealCount { get; set; }
    public List<MenuItemResponseDto> MenuItems { get; set; }
}

File: Client/meal-master/src/app/models/menu-response.dto.ts
// src/app/models/menu-response.dto.ts
import { MenuItemResponseDto } from './menu-item-response.dto';

export interface MenuResponseDto {
  id:string;
  userId: string;
  date: string; // ISO Date string
  totalCalories: number;
  mealCount: number;
  menuItems: MenuItemResponseDto[];
}
